content/doc/administration/requirements/java.adoc

```diff
@@ -9,11 +9,12 @@ There are separate run and job execution requirements for Jenkins installations.
 
 Modern Jenkins versions have the following Java requirements:
 
-* Java 8 is the **ONLY** currently supported runtime environment, both 32-bit and 64-bit versions are supported
+* Java 8 runtime environments, both 32-bit and 64-bit versions are supported
+* Since Jenkins `2.163` footnote:[since `2.155` but it was made easier in `2.163`], Java 11 runtime environments is supported
```

Comment:
for @MRamonLeon 

```suggestion
* Since Jenkins `2.164` footnote:[since `2.155` but it was made easier in `2.163`], Java 11 runtime environments is supported
```

Reply:
To be accurate, 2.164 gather all of the simplifications, I think, although the big one was on 2.163:
```suggestion
* Since Jenkins `2.164` footnote:[since `2.155` but it was made easier in `2.164`], Java 11 runtime environments is supported
```